docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/policies.md

```diff
@@ -0,0 +1,125 @@
+    | A specific API    | `./src/api/<api-name>/policies`                                                                                                                                                                                  |
+    | A specific plugin | `./src/plugins/<plugin-name>/policies`<br/><br/>(see [plugin policies migration](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/update-folder-structure.md#moving-policies) documentation) |
+
+2. (_optional_) Update the policy code. Strapi v4 policies are functions returning `true` or `undefined` to authorize the request.
+
+::: note
+In Strapi v4, depending on whether the policy is applied to REST or GraphQL, the function has access to different contexts:
+
+- Both REST and GraphQL contexts have access to the `is(type: string): boolean`, `type: string`, and `state: object` attributes.
+- The GraphQL context has access to `parent`, `args`, `context`, `info` & `http` (which contains the Koa context) (see [GraphQL customization](http://localhost:8080/developer-docs/latest/plugins/graphql.html#custom-configuration-for-resolvers) documentation).
+- With REST requests, the controller context is merged with the policy context.
+
+:::
+
+Migrating a policy code depends on the code itself and this migration guide can't cover every existing use case. The following examples cover some common use cases of v4 policies working with both REST and GraphQL APIs or specifically with one of these APIs. These examples can be used for global, API-related, or plugin-related policies.
+
+::: details Example: Authorize only logged in users (REST and GraphQL)
+    
+The following v3 policy authorizes only logged in users:
+  
+  ```jsx
+  module.exports = async (ctx, next) => {
+    if (ctx.state.user) {
+      // Go to next policy or will reach the controller's action.
+      return await next();
+    }
+   
+    ctx.unauthorized(`You're not logged in!`);
+  };
+  ```
+  
+To update this policy to v4, replace it with the following code:
+  
+  ```jsx
+  module.exports = (context, config, { strapi }) => {
+    if (context.state.user) { // if a session is open
+      // go to next policy or reach the controller's action
+      return true;
+    }
+  
+    return false; // If you return nothing, Strapi considers you didn't want to block the request and will let it pass
+  };
+  ```
+
+The v4 policy uses the `context.state` variable, accessible to both REST and GraphQL.
+
+:::
+
+::: details Example: Authorize only requests with an id filter (REST and GraphQL)
+
+The following v4 policy only authorizes requests if an `id` filter is provided. The policy uses the `context.is()` method to check if the policy is being tested for a Koa controller or a GraphQL resolver:
+
+```jsx
```

Comment:
While this is valid, it's a really odd way to return the policy. Not something I'd recommend

It's fine logically as, I think :thinking: